src/lib.rs

```diff
@@ -21,12 +21,58 @@ use std::fmt::Write;
 use tree_sitter::{Language, Node};
 
 pub struct FormatOptions {
+    /// Number of spaces used for one level of indentation
     pub indentation: usize,
+    /// Wether to align prefix IRIs,
+    /// or use a single space after the ':'.
+    pub align_prefix_iris: bool,
+    /// Wether to re-indent comments,
+    /// ensuring they all have exactly one space after the '#'.
+    pub unify_comment_indents: bool,
+    /// Wether to sort subjects
+    pub sort_subjects: bool,
+    /// Wether to sort predicates within a subject
+    pub sort_predicates: bool,
+    /// Wether to sort objects withing one subject-predicate pair
+    pub sort_objects: bool,
+    /// Wether a subjects finalizing dot should be on a new line,
+    /// or on the same line as the last object
+    pub subject_dot_on_new_line: bool,
+    /// Whether the first predicate of a subject should be on a new line,
+    /// or on the same line as the subject
+    pub first_predicate_on_new_line: bool,
+    /// Wether to move the first object within one subject-predicate pair onto a new line,
+    /// or on the same line as the predicate
+    pub first_object_on_new_line: bool,
+    /// Wether to move the a single object (within one subject-predicate pair) onto a new line,
+    /// or to keep it on the same line as the predicate
+    pub single_object_on_new_line: bool,
+    /// Wether to put each objects within one subject-predicate pair on a separate line
+    pub objects_on_separate_lines: bool,
+    /// Wether to put each collection item on a separate line.
+    /// See <https://www.w3.org/TR/rdf12-turtle/#collections>.
+    pub collection_item_on_new_line: bool,
+    /// Wether to put each predicate within a blank-node on a separate line.
+    pub blank_node_predicates_on_separate_lines: bool,
```

Comment:
What about merging all these "new_line" options into a single `new_lines_for_easy_diff` option?

Reply:
For me that is fine, yes.

Reply:
I merged all except one of these (new-line) config options into one now.
The left over separate one is:

```rust
    /// Wether to move a single/lone object (within one subject-predicate pair) onto a new line,
    /// or to keep it on the same line as the predicate.
    pub single_object_on_new_line: bool,
```

I am not sure about this one.
It is a bit extreme from the point of view of a human consumer of the Turtle, but could still potentially make sense from a machine-readability pint of view (though arguably less then the other new-lines).